packages/common/test/services/logger.service.spec.ts

```diff
@@ -260,12 +260,13 @@ describe('Logger', () => {
 
   describe('[instance methods]', () => {
     describe('when the default logger is used', () => {
-      const logger = new Logger();
+      let logger = new Logger();
```

Comment:
```suggestion
      let logger: Logger;
```

as this will be instantianted on `beforeEach`

Reply:
Applied